portal/src/graphql/adminapi/UserDetailsAdminActions.tsx

```diff
@@ -0,0 +1,166 @@
+import { Label, Text } from "@fluentui/react";
+import { FormattedMessage } from "@oursky/react-messageformat";
+import React from "react";
+import cn from "classnames";
+import styles from "./UserDetailsAdminActions.module.css";
+import ListCellLayout from "../../ListCellLayout";
+import DefaultButton from "../../DefaultButton";
+import { useSystemConfig } from "../../context/SystemConfigContext";
+
+interface DisableUserCellProps {
+  isDisabled: boolean;
+}
+
+interface AnonymizeUserCellProps {
+  isScheduledAnonymization: boolean;
+}
+
+interface RemoveUserCellProps {
+  isScheduledRemoval: boolean;
+}
+
+const DisableUserCell: React.VFC<DisableUserCellProps> =
+  function DisableUserCell(props) {
+    const { themes } = useSystemConfig();
+    const { isDisabled } = props;
+    return (
+      <ListCellLayout
+        className={cn(
+          styles.cell,
+          styles.actionCell,
+          styles["cell--not-first"]
+        )}
+      >
+        <Text className={cn(styles.cellLabel, styles.actionCellLabel)}>
+          <FormattedMessage id="UserDetailsAdminActions.disable-user.title" />
+        </Text>
+        <Text className={cn(styles.actionCellBody, styles.actionCellLast)}>
+          <FormattedMessage id="UserDetailsAdminActions.disable-user.body" />
+        </Text>
+        {isDisabled ? (
+          <DefaultButton
+            theme={themes.actionButton}
+            className={cn(styles.actionCellActionButton, styles.actionBorder)}
```

Comment:
actionCellActionButton modifies padding, which is considered as the implementation details of DefaultButton. You use `styles` to override padding. In general, you use `className` to position the component, and use specific prop, like `variant`, `styles` to customize the Fluent component.